playbooks/templates/rax-maas/ironic_capacity_check.yaml.j2

```diff
@@ -0,0 +1,23 @@
+{% set label = "ironic_capacity_check" %}
+{% set check_name = label+'--'+inventory_hostname %}
+type        : agent.plugin
+label       : "{{ check_name }}"
+period      : "{{ maas_check_period_override[label] | default(maas_check_period) }}"
+timeout     : "{{ maas_check_timeout_override[label] | default(maas_check_timeout) }}"
+disabled    : "{{ (check_name | match(maas_excluded_checks_regex)) | ternary('true', 'false') }}"
+details     :
+    file    : run_plugin_in_venv.sh
+    args    : ["{{ maas_plugin_dir }}/ironic_capacity_check.py", "{{ ansible_host }}"]
+alarms      :
+    ironic_capacity :
+        label                   : ironic_capacity--{{ inventory_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan_override[label] | default(maas_notification_plan) }}"
+        disabled                : {{ (('ironic_capacity--'+inventory_hostname) | match(maas_excluded_alarms_regex)) | ternary('true', 'false') }}
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (percentage(metric["ironic_up_nodes_count"] , metric["ironic_total_nodes_count"]) <= {{ maas_capacity_percent_critical_threshold }}) {
```

Comment:
What I meant with my previous comment was that `ironic_maint_nodes_count` gets output from the plugin as a metric but nothing in here consumes it—we only look at up versus total. It seems like maybe we shouldn't be reporting that as a metric.